File: sample/dist2/java/src/main/java/org/yanex/hello/IDeprecatedInterface.java
package org.yanex.hello;

/**
 * @deprecated useless interface
 */
@javax.annotation.Generated(value = "jsii-pacmak/0.19.0 (build 5d7824d)", date = "2019-10-28T18:25:08.409Z")
@Deprecated
@software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
public interface IDeprecatedInterface extends software.amazon.jsii.JsiiSerializable {

    /**
     * @deprecated could be better
     */
    @Deprecated
    @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
    java.lang.Number getMutableProperty();

    /**
     * @deprecated could be better
     */
    void setMutableProperty(final java.lang.Number value);

    /**
     * @deprecated services no purpose
     */
    @Deprecated
    @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
    void method();

    /**
     * A proxy class which represents a concrete javascript instance of this type.
     */
    final static class Jsii$Proxy extends software.amazon.jsii.JsiiObject implements org.yanex.hello.IDeprecatedInterface {
        protected Jsii$Proxy(final software.amazon.jsii.JsiiObjectRef objRef) {
            super(software.amazon.jsii.JsiiObject.InitializationMode.JSII);
            this.setObjRef(objRef);
        }

        /**
         * @deprecated could be better
         */
        @Override
        @Deprecated
        @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
        public java.lang.Number getMutableProperty() {
            return this.jsiiGet("mutableProperty", java.lang.Number.class);
        }

        /**
         * @deprecated could be better
         */
        @Override
        @Deprecated
        @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
        public void setMutableProperty(final java.lang.Number value) {
            this.jsiiSet("mutableProperty", value);
        }

        /**
         * @deprecated services no purpose
         */
        @Deprecated
        @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
        @Override
        public void method() {
            this.jsiiCall("method", Void.class);
        }
    }
}

File: sample/dist2/kotlin/src/main/kotlin/hello/IDeprecatedInterface.kt
package org.yanex.hello

@javax.annotation.Generated(value = "jsii-pacmak/0.19.0 (build 5d7824d)", date = "2019-10-29T17:33:08.483Z")
@Deprecated(useless interface)
@software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
interface IDeprecatedInterface: software.amazon.jsii.JsiiSerializable {
    @Deprecated(could be better)
    @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
    var mutableProperty: kotlin.Number?

    @Deprecated(services no purpose)
    @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
    fun method() {
        this.jsiiCall("method", kotlin.Any::class.java)
    }

    internal class `Jsii$Proxy`: software.amazon.jsii.JsiiObject, org.yanex.hello.IDeprecatedInterface {
        @Deprecated(could be better)
        @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
        override var mutableProperty: kotlin.Number?

        protected constructor(objRef: software.amazon.jsii.JsiiObjectRef) : super(software.amazon.jsii.JsiiObject.InitializationMode.JSII) {
            this.objRef = objRef
            this.mutableProperty = this.jsiiCall("mutableProperty", kotlin.Number::class.java)
        }

        @Deprecated(services no purpose)
        @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Deprecated)
        fun method() {
            this.jsiiCall("method", kotlin.Any::class.java)
        }
    }
}
